### PR TITLE
Add slides on GitHub API

### LINKED ISSUE #16: Add content on GitHub API
See https://hbz.github.io/swib16-workshop/#/17

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve #16 

Starts at https://hbz.github.io/swib16-workshop/#/17

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -717,11 +717,87 @@
...
+						<p><span class="fragment"><a href="https://stedolan.github.io/jq/" target="_blank">https://stedolan.github.io/jq/</a></span></p>
+						<p><span class="fragment"><a href="https://jqplay.org/">https://jqplay.org/</a></span></p>
+					</section>
+					<section>
+<h2>Exercise 14: jq basics</h2>
+<pre><code class="bash"># We start with cURL again:
+curl 'https://api.github.com/repos/hbz/swib16-workshop/commits'
+
+# We pipe the JSON to the jq format
+curl 'https://api.github.com/repos/hbz/swib16-workshop/commits' | jq
```
Comment (acka47) [inline index.html:786]: While `curl 'https://api.github.com/repos/hbz/swib16-workshop/commits'` is fine, this currenrtly doesn't work for me. Output:

```
$ curl 'https://api.github.com/repos/hbz/swib16-workshop/commits' | jq
  % Total    % Received % X
jq - commandline JSON processor [version 1.3]
ferUsage: jq [options] <jq filter> [file...]

d  AvFor a description of the command line options and
ehow to write jq filters (and why you might want to)
rsee the jq manpage, or the online documentation at
ahttp://stedolan.github.com/jq

ge Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  4  103k    4  4360    0     0   5689      0  0:00:18 --:--:--  0:00:18  5684
curl: (23) Failed writing body (1106 != 1370)
```

**Comment by fsteeg** [comment]
Added some introductory slides on git and version control: https://hbz.github.io/swib16-workshop/#/4

File: index.html (hunk 1 above)
Comment (fsteeg) [reply index.html:786]: I just tested it here, pasting your line, and it works.

Perhaps older jq versions don't support calling `jq` without parameters?

Mine is 1.5.1, yours 1.3.

**Comment by fsteeg** [comment]
Added a slide on data licenses (vs. software licenses): https://hbz.github.io/swib16-workshop/#/17/2

File: index.html (hunk 1 above)
Comment (acka47) [reply index.html:786]: 1.3 is what I got installing it via the package manager...

**Comment by fsteeg** [comment]
And a slide on GitHub wikis: https://hbz.github.io/swib16-workshop/#/7/5

**Pull request merged**